gcs_jupyter_plugin/controllers/gcs.py

```diff
@@ -126,24 +125,26 @@ async def get(self):
         try:
```

Comment:
This handler must be changed to require a POST request since it modifies state on the server (by downloading a file)